tslint.json

```diff
@@ -0,0 +1,35 @@
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:recommended",
+    "tslint-react",
+    "tslint-config-airbnb",
+    "tslint-plugin-prettier",
+    "tslint-config-prettier"
+  ],
+  "jsRules": true,
+  "rules": {
+    "prettier": true,
+    "import-name": false,
+    "variable-name": false,
+    "member-access": false,
+    "interface-name": false,
+    "object-literal-sort-keys": false,
+    "jsx-boolean-value": false,
+    "jsx-no-lambda": false,
+    "ban-types": {
+      "options": [
+        ["Object", "Avoid using the `Object` type. Did you mean `object`?"],
+        ["Boolean", "Avoid using the `Boolean` type. Did you mean `boolean`?"],
+        ["Number", "Avoid using the `Number` type. Did you mean `number`?"],
+        ["String", "Avoid using the `String` type. Did you mean `string`?"],
+        ["Symbol", "Avoid using the `Symbol` type. Did you mean `symbol`?"]
+      ]
+    },
+    "prefer-array-literal": {
+      "options": { "allow-type-parameters": true }
+    },
+    "no-console": { "severity": "warning" }
+  },
+  "rulesDirectory": []
+}
```

Comment:
`tslint` is getting [deprecated](https://github.com/palantir/tslint#tslint) in favor of `eslint`; we recently set up new projects like [berth-reservation-admin](https://github.com/City-of-Helsinki/berth-reservations-admin) and [kukku-ui](https://github.com/City-of-Helsinki/kukkuu-ui/) if you'd like to use them as references.

Reply:
Alright, will change this to `eslint`. 👍

Reply:
Done.